relayer-engine/src/storage/storage.ts

```diff
@@ -32,29 +32,70 @@ const CHECK_REQUEUED_WORKFLOWS_LOCK = "__isRequeueJobRunning";
 const CHECK_STALE_ACTIVE_WORKFLOWS_LOCK = "_isStaleWorkflowsJobRunning";
 const EMITTER_KEY = "__emitter";
 
+const CONSTANTS = {
```

Comment:
I'm worried it's easy to reach for the bare constant instead of `this.constants.X` with this setup and there will be nasty bugs later. Can we move the constant definitions and the namespace mapping to a new file (e.g. `dbConstants.ts` or smt) and just expose the mapping fn?